package.json

```diff
@@ -17,15 +17,18 @@
   },
   "dependencies": {
     "brace": "^0.8.0",
+    "classnames": "^2.2.5",
```

Comment:
Apparently this is an open issue:
https://github.com/roylee0704/react-flexbox-grid/issues/49